app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/NutritionProductFragment.java

```diff
@@ -74,7 +75,8 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         }
 
         if (fat == null && salt == null && saturatedFat == null && sugars == null) {
-            levelItem.add(new NutrientLevelItem(getString(R.string.txtNoData), "", "", R.drawable.error_image));
+            textNutrientTxt.append(" "+getString(R.string.txtNoData));
```

Comment:
Well it's pity that this line is not correctly formatted, missing spaces around `+`

Reply:
Thanks for pointing it out.I missed it.